optimum/exporters/openvino/model_configs.py

```diff
@@ -1804,8 +1804,9 @@ def __init__(
         normalized_config: NormalizedVisionConfig,
         batch_size: int = DEFAULT_DUMMY_SHAPES["batch_size"],
         num_channels: int = DEFAULT_DUMMY_SHAPES["num_channels"],
-        width: int = DEFAULT_DUMMY_SHAPES["width"],
-        height: int = DEFAULT_DUMMY_SHAPES["height"],
+        width: int = DEFAULT_DUMMY_SHAPES["width"] // 4,
```

Comment:
This unfortunately doesn't work as defaults are set in the optimum code and this will not change the behaviour

Reply:
@echarlaix do you know where exactly we can change these values for a specific architecture like Flux ?
I thought dummy inputs generators will use the defaults in their signature by default unless a user passes input shapes.